cmd/kube-controller-manager/app/options/persistentvolumebindercontroller.go

```diff
@@ -43,6 +48,8 @@ func (o *PersistentVolumeBinderControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.BoolVar(&o.VolumeConfiguration.EnableHostPathProvisioning, "enable-hostpath-provisioner", o.VolumeConfiguration.EnableHostPathProvisioning, "Enable HostPath PV provisioning when running without a cloud provider. This allows testing and development of provisioning features.  HostPath provisioning is not supported in any way, won't work in a multi-node cluster, and should not be used for anything other than testing or development.")
 	fs.BoolVar(&o.VolumeConfiguration.EnableDynamicProvisioning, "enable-dynamic-provisioning", o.VolumeConfiguration.EnableDynamicProvisioning, "Enable dynamic provisioning for environments that support it.")
 	fs.StringVar(&o.VolumeConfiguration.FlexVolumePluginDir, "flex-volume-plugin-dir", o.VolumeConfiguration.FlexVolumePluginDir, "Full path of the directory in which the flex volume plugin should search for additional third party volume plugins.")
+	fs.StringVar(&o.DialHostCIDRDenylist, "dial-host-cidr-denylist", o.DialHostCIDRDenylist, "A comma-separated list of CIDR ranges to avoid dialing from volume plugins, or `none'. If `none', nothing is filtered. Otherwise, non-proxyable hosts are filtered in addition to any specified CIDR ranges.")
```

Comment:
slight preference for making `DialHostCIDRDenylist` a `[]string` and using `StringSliceVar` which handles the comma-separated parsing for you

Reply:
Using it as a string makes it slightly cleaner to parse the "none" option, which shouldn't be one option of many. Conceptually, this is a flag of type {none, comma-separated-CIDRs} so it isn't exactly a StringSliceVar.

Reply:
If I understand correctly, we're logically populating two things in the filteroptions struct:
* DenyNonproxyableHosts bool
* DialHostCIDRDenylist []string

For backports, we want to default those to `false, []`

Going forward, we want to default those to `true, []`

Trying to address both of those values in a single flag is pretty confusing. If the `DenyNonproxyableHosts` field was tied to its own flag (which defaulted to `false` in backports and `true` going forward), would that be easier to reason about? That would let the CIDR deny-list be a simple list with no special "none" semantics.

Reply:
Yeah, that's a good idea. Done.